client/my-sites/stats/stats-date-picker/index.jsx

```diff
@@ -125,20 +127,32 @@ class StatsDatePicker extends Component {
 	}
 
 	render() {
-		const { summary, translate, query, showQueryDate } = this.props;
+		const { summary, translate, query, showQueryDate, isActivity } = this.props;
 		const isSummarizeQuery = get( query, 'summarize' );
 
-		const sectionTitle = translate( 'Stats for {{period/}}', {
-			components: {
-				period: (
-					<span className="period">
-						<span className="date">{ isSummarizeQuery ? this.dateForSummarize() : this.dateForDisplay() }</span>
-					</span>
-				)
-			},
-			context: 'Stats: Main stats page heading',
-			comment: 'Example: "Stats for December 7", "Stats for December 8 - December 14", "Stats for December", "Stats for 2014"'
-		} );
+		const sectionTitle = isActivity
+			? translate( 'Activity for {{period/}}', {
+				components: {
+					period: (
+						<span className="period">
+							<span className="date">{ isSummarizeQuery ? this.dateForSummarize() : this.dateForDisplay() }</span>
+						</span>
+					)
+				},
+				context: 'Activity Log: Main activity section heading',
```

Comment:
It is likely that we don't need `context` here. [`context`](https://codex.wordpress.org/I18n_for_WordPress_Developers#Disambiguation_by_context) is only necessary when two phrases that are the same in english may have different meanings in some languages.

Information about the string can go in `comment`.